source/_integrations/media_extractor.markdown

```diff
@@ -2,7 +2,7 @@
 title: Media Extractor
 description: Instructions on how to integrate the Media Extractor into Home Assistant.
```

Comment:
```suggestion
title: Media extractor
description: Instructions on how to integrate the Media extractor into Home Assistant.
```

Reply:
The reason I didn't change this is because I thought Media Extractor is a name, not generic.